lib/administrate/field/paperclip.rb

```diff
@@ -11,6 +11,10 @@ def style(size = big_style)
         data.try(:url, size) || ''
       end
 
+      def blank?
+        data.blank?
+      end
```

Comment:
The Travis build is currently failing because of Rubocop. Can we delegate this so Rubocop is happy?
http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Rails/Delegate
Thanks!